### DISCUSSION TITLE
Issue setting up connecting integration to pfSense 23.09.1

### DISCUSSION THREAD

**Discussion by mnarusiewicz** [travisghansen/hass-pfsense#180] [Q&A]
Looking from some guidance on how to connect hass-pfsense.  I'm getting the error "User has insufficient privileges (System - HA node sync)".  I created an admin user account, no luck.  I then under System -> High Availability, section "Configuration Synchronization Settings (XMLRPC Sync)" filed in the newly created admin account and password into the Remote System User name and password, no Luck.  Then filled in the Synchronize Config to IP, again no luck.  Besides a admin account, which I agree is not a great solution, it's needed for the API connect and data requested, exactly what privilege is needs to be granted to the user?  Thank you in advance. 

<img width="963" alt="Screenshot 2023-12-29 at 3 49 30 PM" src="https://github.com/travisghansen/hass-pfsense/assets/25290105/82352118-9e13-4c30-bc8f-40382378abed">

**Comment by ice-92** [comment]
Did you figure this out? I have the exact same question. There doesn't seem to be any documentation about this, that I've found...

**Reply by mnarusiewicz** [reply]
Nope, never did get it resolved.
